board/freescale/imx8mm_evk/spl.c

```diff
@@ -2,7 +2,7 @@
 /*
  * Copyright 2019 NXP
  */
-
+#define DEBUG 1
```

Comment:
This from the last commit, which I added just for testing (will be dropped in non-RFC PR)